kroxylicious-operator/pom.xml

```diff
@@ -357,7 +357,7 @@
                             <!-- Make target/kroxylicious-operator-${version}.zip with the test contents from target/packaged -->
                             <execution>
                                 <id>operator-asset</id>
-                                <phase>install</phase><!-- Note this execution happens after the following plugins bound to the package phase! -->
+                                <phase>post-integration-test</phase><!-- Note this execution happens after the following plugins bound to the package phase! -->
```

Comment:
so the order is now:

1. package - assembly plugin execution 1 `-app` directory
2. package - jbang plugin, rename and package up CRDs into target/packaged/install
3. post-integration-test - assembly plugin execution 2 the operator distribution zip/tarball
4. verify - sign the archives

?

Reply:
Thats the intent, yes.